app/controllers/application_controller.rb

```diff
@@ -26,7 +26,7 @@ def redirect_if_no_faction
   end
 
   def call_police(player)
-    if Npc.where(npc_type: 'police', target: player.id).empty?
+    if !Npc.police.targeting_user(player).exists?
```

Comment:
I think `unless` would be better here

Reply:
you are correct.  `unless` is fine -- but i have found that it causes more readability issues than it provides... whereas `if !something` reads as `if not something`

e.g. `unless this && that` is not very readable

Reply:
Problem is I use `unless` also in some other places in this code. I think it would be better if we keep it consistent.

Reply:
understood...